klvdata/misb0601.py

```diff
@@ -851,7 +851,7 @@ class PlatformMagneticHeading(MappedElementParser):
 @UASLocalMetadataSet.add_parser
 class UASLSVersionNumber(StringElementParser):
     key = b'\x41'
-    TAG = 64
+    TAG = 65

```

Comment:
+1

Reply:
Is the change from StringElementParser to BytesElementParser related to the change in commit ef190afe2a237cc6f52a8a78c18888a0ab59df94? 

What happens if you use the MappedElementParser with _domain=(0, 2**8-1) and _range=(0, 255)?